cpg-language-python/src/test/kotlin/de/fraunhofer/aisec/cpg/frontends/python/PythonFrontendTest.kt

```diff
@@ -104,7 +104,7 @@ class PythonFrontendTest : BaseTest() {
         val p = tu.namespaces["function"]
         assertNotNull(p)
 
-        val foo = p.declarations.first() as? FunctionDeclaration
+        val foo = p.declarations.firstOrNull() as? FunctionDeclaration
         assertNotNull(foo)
```

Comment:
assertIs

Reply:
I wasn't planning on having all this improvements here (and opened separate issues), but it makes sense to have all those minor improvements in one PR -> WIP